addons/l10n_it_edi_extended/models/account_move.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+from collections import namedtuple
+from odoo import models
+
+_logger = logging.getLogger(__name__)
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    def _l10n_it_edi_fatturapa_template(self, is_simplified=False):
+        if not is_simplified:
+            return 'l10n_it_edi_extended.account_invoice_it_FatturaPA_export_extended'
+        return super()._l10n_it_edi_fatturapa_template(is_simplified)
```

Comment:
```suggestion
```
No need as the original view is changed by the inheritance.

Reply:
It's not a view. It's a template, manually rendered.
I don't know how to inherit the template and maintain the name, especially the module prefix 
`<module>.<view_name>`

See: [/home/odoo/work/odoo/addons/l10n_it_edi/models/account_invoice.py](https://github.com/odoo/odoo/pull/96930/files#diff-4073c014f2f326e571d34af24c59b9361fc7b0015dbfc3dc55e0f076b8d3f6a2R328)